.superset/setup.local.sh

```
@@ -0,0 +1,228 @@
+#!/usr/bin/env bash
+# Local contributor setup — provisions a fully local Superset workspace with no
+# real credentials: fake env, local Postgres (docker), migrations, a seeded dev
+# account, dynamic ports, and Electric. Mirrors setup.sh but replaces Neon and
+# the "copy my signed-in data" seeds with a self-contained local stack.
+set -uo pipefail
+
+SUPERSET_SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+REPO_ROOT="$(cd "$SUPERSET_SCRIPT_DIR/.." && pwd)"
+cd "$REPO_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)